src/index.html

```diff
@@ -12,6 +12,7 @@
       href="https://fonts.googleapis.com/css2?family=Public+Sans:ital,wght@0,100..900;1,100..900&display=swap"
       rel="stylesheet"
     />
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet">
```

Comment:
there is no need for fontAwesome we use bootstrap icons https://ng-icons.github.io/ng-icons/#/browse-icons